source/skins/simple/OptionsSkin.js

```diff
@@ -116,6 +119,14 @@ export const OptionsSkin = (props: Props) => {
     return option;
   };
 
+  const getScrollBarHeight = (): number => {
+    if (!options.length) return optionHeight;
+    if (optionsMaxHeight < options.length * optionHeight) {
+      return optionsMaxHeight;
+    }
+    return options.length * optionHeight;
+  };
+
```

Comment:
@DominikGuzei 
Since you last reviewed this PR, I added `optionHeight` as a configurable prop as you suggested, but also fixed the scrollbar height calculation. This function is the addition/fix I made which uses `optionHeight`. I think this thing is working perfectly now... I hope :-)